### PR TITLE
fix error when ListView is empty

### PR DESCRIPTION
**Pull request by enhhh**
size_t is an unsigned int ,when ListView has no item,total height will be a max value.and cause jumpToTop and other scroll functions error

### CODE REVIEW

File: cocos/ui/UIListView.cpp [hunk 1]
```diff
@@ -93,7 +93,7 @@ void ListView::updateInnerContainerSize()
         case Direction::VERTICAL:
         {
             size_t length = _items.size();
-            float totalHeight = (length - 1) * _itemsMargin;
+            float totalHeight = ((float)length - 1) * _itemsMargin;
```
Comment (zilongshanren) [inline cocos/ui/UIListView.cpp]: _itemsMargin is float, so the (length-1) \* _itemsMargin would also be float, right?
Why should we add a extra cast here?

Comment (enhhh) [inline cocos/ui/UIListView.cpp]: length is unsigned int , if length is 0, (length - 1) will be a max value, so if _itemsMargin is not 0,totalHeight will be a large value

.

Comment (zilongshanren) [inline cocos/ui/UIListView.cpp]: Thanks @enhhh 
size_t is evil here. we should change it with `int` or `ssize_t`.
Thoughts?

Comment (enhhh) [inline cocos/ui/UIListView.cpp]: change `size_t` with `int` or `ssize_t` is a good idea. :)

**Pull request closed without merging**